### ISSUE TITLE
Touch Event Not working on Google Chrome- Android

### ISSUE THREAD

**Issue by mukeshmishra997** [opensourceBIM/BIMsurfer-before2019#157]
I tried to try to open BimSurfer on Google Chrome(Android). But, I am unable to rotate the 3d view and Pinch to Zoom.

**Comment by xeolabs** [comment]
Yeah there are no touch handlers in the [camera controls](https://github.com/opensourceBIM/BIMsurfer/blob/V2/bimsurfer/src/xeoViewer/controls/bimCameraControl.js) yet. I'll be adding them soon.

**Comment by mukeshmishra997** [comment]
Thank you
